relay-wstring/src/lib.rs

```diff
@@ -279,7 +282,7 @@ impl<'a> Iterator for WStrChars<'a> {
     fn count(self) -> usize {
         // No need to fully construct all characters
         self.chunks
-            .filter(|bb| !is_trailing_surrogate(u16::copy_from_le_bytes(bb)))
+            .filter(|bb| !is_trailing_surrogate(u16::copy_from_le_bytes(bb).unwrap()))
```

Comment:
Would this be better?

```suggestion
            .filter_map(|chunk| u16::copy_from_le_bytes(chunk))
            .filter(|unit| !is_trailing_surrogate(unit))
```

Reply:
heh, i did consider that.  i just feels so ugly and confusing because you're not actually filter_mapping, you're just unpacking an option which you know can be unpacked.  Same with the other places of this commit that add `?`, in a way `.unwrap()` would have been clearer there.

Reply:
I mean, it's up to you. If you know that it can't go wrong, I think it's fair to use an infallible API.